front/migrations/db/migration_93.sql

```diff
@@ -0,0 +1,4 @@
+-- Migration created on Sep 26, 2024
+DROP INDEX IF EXISTS "labs_transcripts_histories_file_id";
+DROP INDEX IF EXISTS "labs_transcripts_histories_file_id_configuration_id";
+CREATE UNIQUE INDEX "labs_transcripts_histories_file_id_configuration_id" ON "labs_transcripts_histories" ("fileId", "configurationId");
```

Comment:
Proper way to do it:

```suggestion
DROP INDEX IF EXISTS "labs_transcripts_histories_file_id_configuration_id";
CREATE UNIQUE INDEX "labs_transcripts_histories_file_id_configuration_id" ON "labs_transcripts_histories" ("fileId", "configurationId");
DROP INDEX IF EXISTS "labs_transcripts_histories_file_id";
```